routing/src/display.rs

```diff
@@ -723,22 +731,6 @@ impl Display for Fib {
         Ok(())
     }
 }
-impl Display for FibTable {
```

Comment:
Even if the fields are owned, aren't they properly protected by left-right?  And if so, can't this stay as is?